msal/src/main/java/com/microsoft/identity/client/internal/controllers/BrokerAccountManagerStrategy.java

```diff
@@ -212,11 +212,9 @@ public Bundle getRequestBundle(SilentTokenCommandParameters parameters) {
                                 parameters,
                                 negotiatedBrokerProtocolVersion
                         );
-                        requestBundle.putString(AuthenticationConstants.Broker.BROKER_ACCOUNT_MANAGER_OPERATION_KEY,
-                                AuthenticationConstants.BrokerAccountManagerOperation.ACQUIRE_TOKEN_SILENT);
                         requestBundle.putString(
-                                AuthenticationConstants.Broker.NEGOTIATED_BP_VERSION_KEY,
```

Comment:
Are we no longer sending this?

Reply:
We do, but i moved the logic to `MsalBrokerRequestAdapter` where the `bundle` is constructed to avoid duplicating in all strategies